scriptmodules/supplementary/emulationstation.sh

```diff
@@ -257,7 +257,7 @@ if [[ "\$(uname --machine)" != *86* ]]; then
 fi
 
 # use SDL2 wayland video driver if wayland session is detected
-[[ "$XDG_SESSION_TYPE" == "wayland" ]] && export SDL_VIDEODRIVER=wayland
+[[ "\$WAYLAND_DISPLAY" == *wayland* ]] && export SDL_VIDEODRIVER=wayland
```

Comment:
Is the first asterisk needed? (As you say the variable starts with wayland) Maybe it's just enough to see if this variable contains something if it's only set for Wayland?

Reply:
Thanks @joolswills. The asterik is not needed. I remove it.